contrib/collectd/files/collectd

```diff
@@ -0,0 +1,6 @@
+# collectd service
```

Comment:
remove the comment (we don't add them anymore)

Reply:
:+1: